integration/mesos/pom.xml

```diff
@@ -0,0 +1,57 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
This use to be integration/pom.xml prior to this PR. Git just does not understand that this is a rename operation.